src/sql/workbench/parts/connection/common/connectionProviderExtension.ts

```diff
@@ -132,3 +134,19 @@ ExtensionsRegistry.registerExtensionPoint<ConnectionProviderProperties | Connect
 		}
 	}
 });
+
+function resolveIconPath(extension: IExtensionPointUser<ConnectionProviderProperties | ConnectionProviderProperties[]>): void {
+	if (!extension) { return; }
```

Comment:
Minor: if we are copying this code from elsewhere let's mention where, so we can refer back if something changes.

Reply:
I copied some code from Alen Ren's PR (https://github.com/Microsoft/azuredatastudio/compare/master...deployment-workstream-master), which is not merged yet.